Build for NetBSD failed

cargo build for NetBSD is failing

```
   Compiling uucore v0.0.25 (/home/axbannaz/Projects/coreutils/src/uucore)
   Compiling bigdecimal v0.4.0
error[E0308]: mismatched types
   --> src/uucore/src/lib/features/fsext.rs:645:16
    |
632 |     fn block_size(&self) -> i64 {
    |                             --- expected `i64` because of return type
...
645 |         return self.f_bsize;
    |                ^^^^^^^^^^^^ expected `i64`, found `u64`
    |
help: you can convert a `u64` to an `i64` and panic if the converted value doesn't fit
    |
645 |         return self.f_bsize.try_into().unwrap();
    |                            ++++++++++++++++++++

For more information about this error, try `rustc --explain E0308`.
error: could not compile `uucore` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
```

I think `return self.f_bsize as i64` would fix it. tried it locally was successful.

--- a/src/uucore/src/lib/features/fsext.rs
+++ b/src/uucore/src/lib/features/fsext.rs
@@ -643,6 +643,7 @@ impl FsMeta for StatFs {
             not(target_os = "aix"),
             not(target_os = "android"),
             not(target_os = "freebsd"),
+            not(target_os = "netbsd"),
             not(target_os = "openbsd"),
             not(target_os = "illumos"),
             not(target_os = "solaris"),
@@ -654,6 +655,7 @@ impl FsMeta for StatFs {
         #[cfg(all(
             not(target_env = "musl"),
             not(target_os = "freebsd"),
+            not(target_os = "netbsd"),
             not(target_os = "redox"),
             any(
                 target_arch = "s390x",
@@ -668,6 +670,7 @@ impl FsMeta for StatFs {
             target_env = "musl",
             target_os = "aix",
             target_os = "freebsd",
+            target_os = "netbsd",
             target_os = "illumos",
             target_os = "solaris",
             target_os = "redox",

